installer/bundle/bundle_skel_Linux.sh

```diff
@@ -702,29 +581,21 @@ case "$installMode" in
 
     I)
         echo "Installing cross-platform agent ..."
-        if [ "$PLATFORM" = "SunOS" ]; then
-            check_if_pkg_is_installed scx
-            if [ $? -eq 0 ]; then
-                echo "ERROR: SCX package is already installed"
-                cleanup_and_exit 2
-            fi
-        fi
 
         if [ $PROVIDER_ONLY -eq 0 ]; then
             check_if_pkg_is_installed omi
             if [ $? -eq 0 ]; then
-                pkg_upd $OMI_PKG omi
-                # It is acceptable that this fails due to the new omi being 
-                # the same version (or less) than the one currently installed.
-                OMI_EXIT_STATUS=0
+                pkg_upd_list $OMI_PKG omi
+                pkg_upd ${UPD_PKG_QUEUE}
             else
-                pkg_add $OMI_PKG omi
-                OMI_EXIT_STATUS=$?
+                pkg_add_list $OMI_PKG omi
             fi
         fi
 
-        pkg_add $OM_PKG scx
-        SCX_EXIT_STATUS=$?
+        pkg_add_list $OM_PKG scx
+
+        pkg_add ${ADD_PKG_QUEUE}
+        SCX_OMI_EXIT_STATUS=$?
 
         if [ $PROVIDER_ONLY -eq 0 ]; then
```

Comment:
I don't think we actually create a provider-only kit anymore. You can keep it if you want, or you can delete it.

Originally, when planning OMS, the thought was to install SCX itself as part of a mega-bundle (i.e. check in the SCX installation shell bundle from the build into OMS itself, much like we have Apache/MySQL kits checked into SCXcore-osskits). But in the end, I didn't do it this way, so making these "provider-only" changes wasn't actually used.

Now, that said, I think there's support code for that in the configure script. So if you were to remove it, you'd need to remove it from configure, make, and here as well.

It's probably not worth it (remember, I'm fundamentally lazy), but I'll leave that up to you. I wonder if this sort of thing will come in useful later? Probably not, but who knows. I suspect that if it did come up later, it would be in some other form (like not include the MySQL/Apache bundles as part of the kit). But if we did that, it would just involve changing the configure and makefile scripts anyway.